.travis.yml

```diff
@@ -1,16 +1,39 @@
 services:
     - docker
 
+env:
+    global:
+        - TEST_RUNNER_IMAGE="martbab/freeipa-fedora-test-runner:master-latest"
+    matrix:
+        - TESTS_TO_RUN="test_xmlrpc/test_[a-k]*.py"
+        - >
+            TESTS_TO_RUN="test_cmdline
+            test_install
+            test_ipalib
+            test_ipapython
+            test_ipaserver
+            test_pkcs10
+            test_xmlrpc/test_[l-z]*.py"
 before_install:
     - pip install pep8
+    - pip3 install --user git+https://github.com/martbab/ipa-docker-test-runner
```

Comment:
In Travis you can safely install to the global site-packages directory. The image gets destroyed after each run anyway.